app/backend/src/clj/teet/thk/thk_import.clj

```diff
@@ -180,8 +181,14 @@
                            name :activity/name
                            task-id :activity/task-id
                            :as activity} activities
-                          ;; Only process activities here
-                          :when (and id name (nil? task-id))
+                          ;; Only process activities here except 2 with activity_typefk 4006 and 4009
```

Comment:
Could expand comment, why not those activity types?

Reply:
it is a requirement of TEET-1671
Import owners supervision and audit tasks and contracts from thk-teet csv to teet